pkg/api/handlers/compat/compat.go

```diff
@@ -0,0 +1,9 @@
+package compat
+
+import (
+	_ "github.com/containers/podman/v4/pkg/errorhandling" // nolint:golint  imported into pkg for side effect
```

Comment:
Why do you import this in so many places? It should be enough to import it only once.

If you want to make sure it is used for the bindings users it should be enough to import it in `pkg/bindings/connection.go`

Reply:
@Luap99 I imported it into each package

Reply:
I want it to be only imported once, the other imports are just redundant without any value. All bindings users have to use `pkg/bindings/connection.go` so it will be enough to import it there.